jira/resources.py

```diff
@@ -764,6 +764,23 @@ def __eq__(self, other):
         return str(self.name) == str(other.name)
 
 
+class UserGroup(Resource):
```

Comment:
Any reason for not naming it Group instead of UserGroup? I would prefer simple unless we have a good reason.